settings.gradle

```diff
@@ -36,6 +36,7 @@ includeWithFlags ':tomcat8.5',                          'java', 'publish', 'relo
 includeWithFlags ':zookeeper',                          'java', 'publish', 'relocate'
 includeWithFlags ':saml',                               'java', 'publish', 'relocate'
 includeWithFlags ':bucket4j',                           'java', 'publish', 'relocate'
+includeWithFlags ':preprocessor',                       'java', 'publish', 'relocate'
```

Comment:
I'm not sure this is right. I'm not sure what the `relocate` flag does...

Reply:
It's fine. `relocate` relocates the shaded dependencies.

I wonder what would be the best name for this new module. It's currently `armeria-preprocessor`. I see some other projects use the following names:

- `jmh-generator-annprocess`
- `spring-boot-configuration-processor`

`armeria-annotation-processor`? :thinking:

Reply:
I've renamed the package to your suggestion

Reply:
Thanks, @hueneburg